### ISSUE TITLE
Donating

### ISSUE THREAD

**Issue by ChadTaljaardt** [fork-dev/Tracker#325]
Hello, 

Is there a way to become a sponsor or to donate?

Thanks!

**Comment by DanPristupov** [comment]
Hi. We are doing well so far. I develop Fork with my wife, so we don't have much costs :).

Thank you very much for the proposal, I really appreciate that.
Dan

**Comment by lxnk** [comment]
Same here. Awesome job done! And it has to be appreciated.

**Comment by ChristianSteffens** [comment]
Same here: I'd like very much to donate something. I'm using your app daily to do my own work, so it's only fair to give something back. Please take my money 😆

**Comment by DRoet** [comment]
FYI: there is now an option to purchase a license if you really want to support the project, more info here: https://github.com/ForkIssues/Tracker/issues/562#issuecomment-553339374

**Comment by aniforprez** [comment]
Yeah I used to be a CLI kind of guy until I used Sublime Merge which was fairly fast but has the same problem as Sublime Text which is the slow pace of updates, lack of features and extensibility, insanely high pricing etc. It still did the job but the free version nags a fair amount. Then I found Fork which had a free version that never once bothered me and is fast, better than the CLI and extremely user friendly and also extensible with custom commands and with that I completely ditched Sublime Merge. Once the paid option came along as was mentioned by @DRoet I absolutely had to buy a licence. One of the best purchases I've ever made for my dev setup and I recommend it to everyone I know
